### ISSUE TITLE
Doppio scalar field & arithmetic

### ISSUE THREAD

**Issue by cathieyun** [dalek-cryptography/doppio#6]
Unsure if this is necessary (maybe it's enough to just do point operations over the doppio points, without a doppio scalar field?). Also unsure what this scalar field would look like.

**Comment by hdevalence** [comment]
Yes, I think we'll need to implement the scalar field. I'm not sure what the modulus is, we should probably get a hold of the sage script @ebfull used and pull it into this repo so that there's a way to work with the curve in sage.
